cyral/resource_cyral_integration_aws_iam_authn_test.go

```diff
@@ -0,0 +1,212 @@
+package cyral
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+const (
+	arn0 = "arn:aws:iam::421552459763:role/Role1"
+	arn1 = "arn:aws:iam::421552459763:role/Role2"
+)
+
+func TestIntegrationAWSIAMAuthN(t *testing.T) {
+	testCreate := getCreateStep("main_test")
```

Comment:
Lets use the `accTestName` function to make sure resources generated by terraform tests can be identified.
```suggestion
        resourceName := accTestName("integration-aws-iam", "main_test")
	testCreate := getCreateStep(resourceName)
```

Reply:
@ricardorey10 I just noticed that my suggestion above does not make sense. We don't need to use the accTestName for the resourceName if that's only used to identify the terraform resource name. Sorry for that, please feel free to ignore this suggestion.